scipy/stats/_continuous_distns.py

```diff
@@ -3230,7 +3230,20 @@ def _stats(self, a):
         return a, a, 2.0/np.sqrt(a), 6.0/a
 
     def _entropy(self, a):
-        return sc.psi(a)*(1-a) + a + sc.gammaln(a)
+
+        def regular_formula(a):
+            return sc.psi(a) * (1-a) + a + sc.gammaln(a)
+
+        def asymptotic_formula(a):
+            # plug in above formula the expansions:
+            # psi(a) ~ ln(a) - 1/2a - 1/12a^2 + 1/120a^4
+            # gammaln(a) ~ a * ln(a) - a - 1/2 * ln(a) + 1/2 ln(2 * pi) +
+            #              1/12a - 1/130a^3
+            return (0.5 * (1. + np.log(2*np.pi) + np.log(a)) - 1/(3 * a) -
+                    1/(12 * a**2) - 1/(90 * a**3) + 1/(120 * a**4))
```

Comment:
```suggestion
            return (0.5 * (1. + np.log(2*np.pi) + np.log(a)) - 1/(3 * a)
                    - (a**-2)/12 - (a**-3)/90 + (a**-4)/120)
```
Avoids overflow and PEP8 prefers binary operators go on continuation line.